docs/analytics/jenkins.md

```diff
@@ -288,37 +290,34 @@ Troubleshooting
   `manifest.lib_jar` was compiled using a different version of java than what's running on the EMR cluster.
   The easiest way to rebuild the `edx-analytics-hadoop-util.jar` using the correct java version, and the required hadoop
   libraries, is to:
-  1. Launch an EMR cluster using the version of EMR configured for your analytics tasks.
 
-     Alternately, run one of the failing tasks with `export TERMINATE=false` in the environment, and this will leave the
-     EMR cluster running after the job has failed.
+    1. Launch an EMR cluster using the version of EMR configured for your analytics tasks.
+
+        Alternately, run one of the failing tasks with `export TERMINATE=false`
+        in the environment, and this will leave the EMR cluster running after
+        the job has failed.
+
+        Note the EMR Cluster ID for the `aws emr` step below.
+    1. Create a virtualenv, and install awscli:
 
-     Note the EMR Cluster ID for the `aws emr` step below.
-  1. Create a virtualenv, and install awscli:
+            pip install awscli
 
-    ```bash
-    pip install awscli
-    ```
-  1. Create an IAM user and attach the `provision_emr_clusters` policy you created above.
-  1. Using the AWS Access key ID and secret, authenticate your awscli:
+    1. Create an IAM user and attach the `provision_emr_clusters` policy you created above.
+    1. Using the AWS Access key ID and secret, authenticate your awscli:
+
+            aws configure
 
-    ```bash
-    aws configure
-    ```
   1. Shell into the EMR cluster using the `analytics.pem` file:
 
-    ```bash
-    aws emr ssh --cluster-id j-xxxxxxxxxxxx --key-pair-file=analytics.pem
-    ```
+        aws emr ssh --cluster-id j-xxxxxxxxxxxx --key-pair-file=analytics.pem
```

Comment:
Nit: This isn't showing up as a code block in the preview - I think indenting it more might make that happen, but alternatively it could be surrounded by the triple backticks again.